app/src/main/java/io/intrepid/contest/models/EntryBallot.java

```diff
@@ -33,12 +33,4 @@ public boolean isCompletelyScored() {
         }
         return true;
     }
-
-    public float getScoreAverage() {
```

Comment:
Yea this isn't being used anymore now that we are calculating the score within the Entry objects now